test/index.html

```diff
@@ -11,6 +11,7 @@
   <script src="vendor/qunit.js"></script>
   <script src="../underscore-umd.js"></script>
   <script src="qunit-setup.js"></script>
+  <script src="overrides.js"></script>
```

Comment:
The overrides should go before Underscore itself, to ensure that `supportsDataView` is set correctly.